### DISCUSSION TITLE
notes/strong-and-weak-symbol/

### DISCUSSION THREAD

**Discussion by giscus** [x2rs/x2rs.github.io#20] [Giscus Comments]
# notes/strong-and-weak-symbol/

My personal documentation website.

https://x2rs.github.io/notes/strong-and-weak-symbol/

**Comment by x2rs** [comment]
这篇是GPT生成的，主要是当时自己对这方面了解太少，又想写成笔记。
很多内容都没有验证，感觉就这样把GPT生成的东西贴在这有点不负责任。
